data/mods/ssb/random-teams.ts

```diff
@@ -522,6 +522,12 @@ export const ssbSets: SSBSets = {
 		signatureMove: 'Rapid Turn',
 		evs: {atk: 252, def: 4, spe: 252}, nature: 'Jolly',
 	},
+	peapod: {
```

Comment:
thats not your PS name, is it?

Reply:
yeah please match ur current PS username

Reply:
ok